kasa/feature.py

```diff
@@ -107,6 +107,8 @@ class Type(Enum):
     Number = Type.Number
     Choice = Type.Choice
 
+    DEFAULT_MAX = 2**16  # Arbitrary max
```

Comment:
Unrelated changes?

Reply:
This was done as part of the tests added for `lighttransition` to ensure a maximum was set and it wasn't the default.  Seemed less error prone that writing it out again in the test.